ansible/roles/kamailio/files/tls.cfg

```diff
@@ -0,0 +1,13 @@
+#
+# Example Kamailio TLS Configuration File
+#
+# Self signed certificates generation:
+# > openssl req -x509 -newkey rsa:4096 -keyout key.pem -out cert.pem -days 10000 -nodes
+# > sudo chmod -R 755 key.pem  cert.pemsudo chmod -R 755 key.pem  cert.pem
+
+[server:default]
+method = TLSv1
+verify_certificate = no
+require_certificate = no
+private_key = /etc/kamailio/tls/key.pem
+certificate = /etc/kamailio/tls/cert.pem
```

Comment:
Is it normal that those files are not created on the host? Given the previous lines, it seems that TLS is not enabled: an issue/ticket should be opened.